gcm_toolkit/tests/test_interface.py

```diff
@@ -43,15 +43,21 @@ def test_prt_interface(petitradtrans_testdata, all_raw_testdata):
     )
 
     # add cloud properties
-    dsi_clouds = tools.get_models('0')
-    tmp = np.ones((len(dsi_clouds['time']), len(dsi_clouds['lat']),
-                   len(dsi_clouds['lon']), len(dsi_clouds['Z_l'])))
-    dsi_clouds['ClAb'] = (('time', 'lat', 'lon', 'Z_l'), tmp*1e-5)
-    dsi_clouds['ClDs'] = (('time', 'lat', 'lon', 'Z_l'), tmp*1e2)
-    dsi_clouds['ClDr'] = (('time', 'lat', 'lon', 'Z_l'), tmp*1e3)
-    dsi_clouds['ClVf_Test1[s]'] = (('time', 'lat', 'lon', 'Z_l'), tmp*0.5)
-    dsi_clouds['ClVf_Test2[s]'] = (('time', 'lat', 'lon', 'Z_l'), tmp*0.5)
-    tools._replace_model('0', dsi_clouds)
+    dsi_clouds = tools.get_models("0")
+    tmp = np.ones(
+        (
+            len(dsi_clouds["time"]),
+            len(dsi_clouds["lat"]),
+            len(dsi_clouds["lon"]),
+            len(dsi_clouds["Z"]),
+        )
+    )
+    dsi_clouds["ClAb"] = (("time", "lat", "lon", "Z"), tmp * 1e-5)
+    dsi_clouds["ClDs"] = (("time", "lat", "lon", "Z"), tmp * 1e2)
+    dsi_clouds["ClDr"] = (("time", "lat", "lon", "Z"), tmp * 1e3)
+    dsi_clouds["ClVf_Test1[s]"] = (("time", "lat", "lon", "Z"), tmp * 0.5)
+    dsi_clouds["ClVf_Test2[s]"] = (("time", "lat", "lon", "Z"), tmp * 0.5)
+    tools._replace_model("0", dsi_clouds)
```

Comment:
@Kiefersv, could you comment on these changes I had to make? I think the Z_l here is strange?!

Reply:
I double checked and you are indeed right that it should be Z. No idea why it worked before with Z_l.

Reply:
No. Its likely problems related to xarray's master branch. But this looked odd to me.